Source/Libraries/GSF.Core/Data/DataInserter.cs

```diff
@@ -453,22 +451,23 @@ private void ResetAutoIncValues(Table table)
                 {
                     case DatabaseType.SQLServer:
                         resetAutoIncValueSQL = "DBCC CHECKIDENT('" + table.SQLEscapedName + "', RESEED)";
-                        table.Connection.ExecuteNonQuery(resetAutoIncValueSQL, Timeout);
+                        table.Connection.ExecuteNonQuery("DBCC CHECKIDENT('{0}', RESEED)", table.SQLEscapedName, Timeout);
                         break;
                     case DatabaseType.MySQL:
                         resetAutoIncValueSQL = "ALTER TABLE " + table.SQLEscapedName + " AUTO_INCREMENT = 1";
-                        table.Connection.ExecuteNonQuery(resetAutoIncValueSQL, Timeout);
+                        table.Connection.ExecuteNonQuery("ALTER TABLE {0} AUTO_INCREMENT = 1", table.SQLEscapedName, Timeout);
                         break;
                     case DatabaseType.SQLite:
                         resetAutoIncValueSQL = "DELETE FROM sqlite_sequence WHERE name = '" + table.Name + "'";
-                        table.Connection.ExecuteNonQuery(resetAutoIncValueSQL, Timeout);
+                        table.Connection.ExecuteNonQuery("DELETE FROM sqlite_sequence WHERE name = '{0}'", table.Name, Timeout);
                         break;
                     case DatabaseType.PostgreSQL:
                         // The escaping of names here is very deliberate; for certain table names,
                         // it is necessary to escape the table name in the pg_get_serial_sequence() call,
                         // but the call will fail if you attempt to escape the autoIncField name
                         resetAutoIncValueSQL = $"SELECT setval(pg_get_serial_sequence('{table.SQLEscapedName}', '{table.AutoIncField.Name.ToLower()}'), (SELECT MAX({table.AutoIncField.SQLEscapedName}) FROM {table.SQLEscapedName}))";
-                        table.Connection.ExecuteNonQuery(resetAutoIncValueSQL, Timeout);
+                        table.Connection.ExecuteNonQuery("SELECT setval(pg_get_serial_sequence('{0}', '{1}'), (SELECT MAX({2}) FROM {3}))",
```

Comment:
Parameter substitution does not work on Tables

Reply:
updated